README.markdown

```diff
@@ -80,6 +80,9 @@ Usage: `git push-to-trychooser [-t/--tip] PATH_TO_HG_REPO [GIT_REVS]`
 The same as `git push-to-hg`, but also runs the interactive trychooser command
 before pushing the commits to try from the given hg repository.
 
+To use this, you must install the `trychooser` Mercurial extension from Mozilla's
+[version control tools repository](https://hg.mozilla.org/hgcustom/version-control-tools).
```

Comment:
... but there should be. Pretty sure there's a bug on file somewhere to sort out the mess that is the varying trychooser extensions.